modules/check_in/spec/services/travel_claim/client_spec.rb

```diff
@@ -103,12 +108,6 @@
         allow_any_instance_of(Faraday::Connection).to receive(:post).with(anything).and_return(claim_response)
```

Comment:
Could you please see if there is a way to mock the response time and validate `req.options.timeout` changes?